src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  return (
```

Comment:
1. it's a good practice to use object destructing;
2. create a variable to get a person status
```suggestion
export const Person = ({ person }) => {
  const { name, age, sex,  isMarried, partnerName } = person;
  const personStatus = sex === 'f' ? 'husband' : 'wife'
  
  return (
```